docs/references/ios/get-token.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: getToken()
+description: Use Clerk's iOS SDK to retrieve a token for a JWT Template that is defined in the Clerk Dashboard.
+---
+
+# `getToken()`
+
+Retrieves the user's session token for the default Clerk token or the given template.
+
+This method uses a cache so a network request will only be made if the token in memory has expired. The TTL for a Clerk token is one minute.
+
+Tokens can only be generated if the user is signed in.
+
+```swift {{ filename: 'Session' }}
+func getToken(
+  _ options: GetTokenOptions = .init()
+) async throws -> TokenResource?
+```
+
+## Parameters
+
+| Name | Type | Description |
+| - | - | - |
+| `options` | `GetTokenOptions` | Options that can be passed as parameters to the `getToken()` function. |
+
+### `GetTokenOptions`
+
+| Name | Type | Description |
+| - | - | - |
+| `template` | `String?` | The name of the JWT template from the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates) to generate a new token from.<br />For example: 'firebase', 'grafbase', or your custom template's name. |
+| `expirationBuffer` | `Double` | If the cached token will expire within `{expirationBuffer}` seconds, fetch a new token instead. Max is 60 seconds. Defaults to 10 seconds. |
+| `skipCache` | `Bool` | Whether to skip the cache lookup and force a refresh of the token instead. Useful if the token claims are time-sensitive or depend on data that can be updated (e.g. user fields). Defaults to false. |
```

Comment:
use the [`<Properties>` component](https://github.com/clerk/clerk-docs/blob/main/CONTRIBUTING.md#properties-1) here as well